src/components/GridContent.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import img5 from '../img/img5.svg';
+import { Link } from 'react-router-dom';
+import { Grid, Paper, Typography, Button } from '@mui/material';
+
+function GridContent() {
+  return (
+    <Grid
+      container
+      component={Paper}
+      variant="outlined"
+      sx={{ padding: { xs: 2, md: 3 }, mt: 3 }}>
```

Comment:
The `mt` looks a bit extraneous. Isn't going to affect the UserHome page as well?

Reply:
I'll add the margin in the other page.